src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/TableExpression.cs

```diff
@@ -20,7 +20,8 @@ internal class TableExpression : Expression
             Expression normalizedPredicate,
             Expression denormalizedPredicate,
             TableExpressionKind kind,
-            int chainLevel = 0)
+            int chainLevel = 0,
+            Expression denormalizedPredicateOnChainRoot = null)
```

Comment:
It might be helpful to add some parameter documentation to some of these common types, this constructor has 3 expression parameters now :)

Reply:
You are right!